api-extractor/src/definitions/ApiMethod.ts

```diff
@@ -26,14 +26,12 @@ export default class ApiMethod extends ApiMember {
       this.params = [];
       for (const param of methodDeclaration.parameters) {
         const declarationSymbol: ts.Symbol = TypeScriptHelpers.tryGetSymbolForDeclaration(param);
-        const docComment: string = declarationSymbol && this.documentation && this.documentation.paramDocs ?
```

Comment:
Could you run this on sp-core-library and make sure the output JSON doesn't have any diff from the last release?  That would help confirm that we don't need this code.